script/Arcueid/Day 5/05_00_ARC05_1C.txt

```diff
@@ -21,12 +21,12 @@ Fortunately, I have a ton of money.
 [sha:79a85382b633ab382fd47156e46ea43ca1be4dac]{
 -- Page 144, Offset 8934.
 -- 素人考えだろうと、何も無いよりはあった方がいい。何があいつの助けになるか分からないんだから。
-Even if it's an amateurish idea, it's better than coming up empty-handed. You never know what might be of help to that woman.
+Although I'm no expert, my amateur opinion is definitely better than coming up empty-handed. You never know what might be of help to her. 
```

Comment:
Prefer the original for the first sentence.

Reply:
I prefer new version